BitwardenShared/Core/Autofill/Utilities/ASAuthorizationPublicKeyCredentialUserVerificationPreference+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+import AuthenticationServices
+import BitwardenSdk
+
+extension ASAuthorizationPublicKeyCredentialUserVerificationPreference {
+    /// Maps to the Bitwarden Sdk user verification preference object
+    func toSdkUserVerificationPreference() -> BitwardenSdk.Uv {
+        switch self {
+        case ASAuthorizationPublicKeyCredentialUserVerificationPreference.discouraged:
```

Comment:
@matt-livefront I think it's not very straightforward here because `BitwardenSdk.Uv` is an enum and it doesn't have the `init(raw value:)` initializer. I'll do a static init if you agree. Would something like this work?
```swift
extension BitwardenSdk.Uv {
    static func from(preference: ASAuthorizationPublicKeyCredentialUserVerificationPreference) -> BitwardenSdk.Uv {
        switch preference {
        case ASAuthorizationPublicKeyCredentialUserVerificationPreference.discouraged:
            BitwardenSdk.Uv.discouraged
        case ASAuthorizationPublicKeyCredentialUserVerificationPreference.preferred:
            BitwardenSdk.Uv.preferred
        case ASAuthorizationPublicKeyCredentialUserVerificationPreference.required:
            BitwardenSdk.Uv.required
        default:
            BitwardenSdk.Uv.required
        }
    }
}
```

Reply:
You could still add an initializer to `BitwardenSdk.Uv`:
```
extension BitwardenSdk.Uv {
    init(preference: ASAuthorizationPublicKeyCredentialUserVerificationPreference) {
        switch preference {
        case .discouraged:
            self = .discouraged
        case .preferred:
            self = .preferred
        case .required:
            self = .required
        default:
            self = .required
        }
    }
}
```